packages/cli/src/tools/packageManager.ts

```diff
@@ -29,8 +27,8 @@ const packageManagers = {
 
 function configurePackageManager(
   packageNames: Array<string>,
-  options?: Options,
   action: 'install' | 'installDev' | 'installAll' | 'uninstall',
+  options?: Options,
```

Comment:
@Esemesek you ok with that? Makes sense imho

Reply:
Yup. It makes a lot of sense.